mw-webapp/src/component/viewSwitcher/ViewSwitcher.tsx

```diff
@@ -0,0 +1,95 @@
+import clsx from "clsx";
+import {HorizontalContainer} from "src/component/horizontalContainer/HorizontalContainer";
+import {Icon, IconProps, IconSize} from "src/component/icon/Icon";
+import {PositionTooltip} from "src/component/tooltip/PositionTooltip";
+import {Tooltip, TooltipProps} from "src/component/tooltip/Tooltip";
+import {View} from "src/utils/LocalStorageWorker";
+import styles from "src/component/viewSwitcher/ViewSwitcher.module.scss";
+
+/**
+ * View option
+ */
+export interface ViewOption {
+
+  /**
+   * View types
+   */
+  view: View;
+
+  /**
+   * Tooltip's content
+   */
+  tooltipContent: TooltipProps["content"];
```

Comment:
I think it can be just string in this case. Am I wrong? Is there any  other content ?

Reply:
Inside Tooltip props interface it can be string | ReactNode | ReactElement, I would like to have a single place of truth

Reply:
yeah. it will be convenient to use some react nodes in tooltips,

Reply:
Replaced with string | ReactNode | ReactElement